### PR TITLE
Added new Ac

### PR DESCRIPTION
**Pull request by nishantkhimani**

### CODE REVIEW

**Comment by chiragkhimani** [comment]
Please remove duplicate code.

**Comment by nishantkhimani** [comment]
Hey, I just updated the code. Pleaese approve it

File: ARepo/src/Test.java [hunk 1]
```diff
@@ -5,5 +5,6 @@ public static void main(String[] args) {
 		System.out.println("Feature2ZZZZZZ");
 		System.out.println("Feature3 Acc");
 		System.out.println("Feature4 TTTTTTc");
+		System.out.println("Feature5 Acc");
```
Comment (chiragkhimani) [inline ARepo/src/Test.java]: We will hold for feature5 and will release feature6 this time. Please update it

File: ARepo/src/Test.java [hunk 2]
```diff
@@ -5,5 +5,6 @@ public static void main(String[] args) {
 		System.out.println("Feature2ZZZZZZ");
 		System.out.println("Feature3 Acc");
```
Comment (chiragkhimani) [inline ARepo/src/Test.java]: Please remove this line

**Comment by nishantkhimani** [comment]
Please ab to approve kr diyo

**Comment by chiragkhimani** [comment]
Great(y)

**Pull request merged**